recipes/cosmic_profiles/meta.yaml

```diff
@@ -52,9 +53,8 @@ test:
     - cosmic_profiles/tests/test_shapes_ex_script.py
   requires:
     - pytest
-    - pip
   commands:
-    - pip check
```

Comment:
Is `pip check` failing? We recommend to keep it, along with the other tests, unless there is some known error.